fog/overseer/server/src/worker.rs

```diff
@@ -78,14 +78,14 @@ impl OverseerWorker {
 
         Self {
             join_handle,
-            stop_requested,
+            is_enabled,
         }
     }
 
     /// Stop and join the db poll thread
     pub fn stop(&mut self) -> Result<(), ()> {
         if let Some(join_handle) = self.join_handle.take() {
-            self.stop_requested.store(true, Ordering::SeqCst);
+            self.is_enabled.store(false, Ordering::SeqCst);
```

Comment:
Why would this cause the thread to stop?

Reply:
Fixed in most recent commit.